apps/meteor/tests/e2e/administration.spec.ts

```diff
@@ -111,6 +112,47 @@ test.describe.parallel('administration', () => {
 			await expect(poAdmin.tabs.users.inputUserName).toHaveValue(username);
 			await expect(poAdmin.tabs.users.joinDefaultChannels).not.toBeVisible();
 		});
+
+		test.describe('delete user who is last owner of a room', () => {
+			const channelName = faker.string.uuid();
+			let user: IUser;
+			test.beforeAll(async ({ api }) => {
+				const createUserResponse = await api.post('/users.create', {
+					email: faker.internet.email(),
+					name: faker.person.fullName(),
+					password: faker.internet.password(),
+					username: faker.internet.userName(),
+				});
+
+				expect(createUserResponse.status()).toBe(200);
+				user = (await createUserResponse.json()).user;
+
+				// TODO: refactor createChannel utility in order to get channel data when creating
+				const response = await api.post('/channels.create', { name: channelName, members: [user.username] });
+				const { channel } = await response.json();
+
+				await api.post('/channels.addOwner', { roomId: channel._id, username: user.username });
+				await api.post('/channels.removeOwner', { roomId: channel._id, userId: Users.admin.data._id });
+			});
+
+			test('expect to show owner change modal', async ({ page }) => {
```

Comment:
Can we add a test to cover the empty rooms?

Reply:
Done!